traitsui/qt4/helper.py

```diff
@@ -274,4 +261,4 @@ def wrap_text_with_elision(text, font, width, height):
             last_line, QtCore.Qt.ElideRight, width
         )
 
-    return lines
```

Comment:
Unrelated drive-by fix made by my editor (look ma, no hands!). I can revert if preferred.

Reply:
It's fine :)